src/charm.py

```diff
@@ -435,7 +468,7 @@ def _all_config_or_relations(self) -> Any:
     @property
     def _multiple_relations(self) -> bool:
         """Return a boolean indicating if we're related to multiple applications."""
-        return len(self.model.relations["ingress"]) > 1
+        return len(self.model.relations["ingress"]) + len(self.model.relations["nginx-route"]) > 1
```

Comment:
Why not `if self.model.relations["ingress"] or self.model.relations["nginx-route"]`?

Reply:
This requires the relation number greater than 1 rather than greater or equal to 1.

Reply:
Oh, right. Nvm me